docs/TranslationGuide.md

```diff
@@ -25,6 +25,23 @@ following languages with the attached language code:
 * `sv` for Swedish language
 
 
+## Extracting translatable strings
+
+When adding new translatable strings to the GUI, they need to be added to each
+*LANG.json* file. This can be done with the following command:
+
+```
+npm run i18n:extract
+```
+
```

Comment:
Thanks for the documentation. This step should be executed every time a new string is added or changed (in the code).

Furtherdown in this file there is a quote from `package.json`. Should the code in `package.json` be updated? And then also the quote?

Reply:
Good point.

Reply:
> Should the code in package.json be updated?

I'm not sure what you are referring to here, the code has already been updated
https://github.com/zonemaster/zonemaster-gui/blob/fe29580429de92215006cea55087f641d6399711/package.json#L14

> And then also the quote?

I updated the documentation.

Reply:
I found issue #160, do you think we need update the ReleaseProcess-preparation.md document as well?

Reply:
I missed the update of package.json, sorry.

It is probably good that is also done at package build, but then the change will not go back into the repository. A good thing about "null" is that it is visible.

To make this work I think we should have steps like the following:
1. A PR is created
2. A string to be translated is added or modified (could in clear text or just a tag).
3. The command is run to update *.json files
4. en.json is update (if it is a tag)
5. Possibly fr.json and sv.json are updated, but could be postponed
6. The PR is merged
7. Issues for remaining languages are created and those updates are handled in separate PRs.

No, I do not think we should handle that as part of the release process. I will re-review and possibly close the issue. The question is if we need more documentation here in GUI.

Reply:
I closed #160 with a comment.